internal/notify/notifications.lua

```diff
@@ -524,23 +563,6 @@ NOTIFICATIONS_BY_IDX = {
         adv_fn=curry(get_bar, get_blood, get_max_blood, "Blood", COLOR_RED),
         on_click=nil,
     },
```

Comment:
```suggestion
    },
    {
        name='save-reminder',
        desc='Shows a reminder if it has been more than '.. save_time_threshold_mins ..' minute(s) since your last save.',
        default=true,
        fn=get_save_alert,
        on_click=save_popup
    },    
```